registry/storage/driver/s3-aws/s3.go

```diff
@@ -834,8 +834,7 @@ ListLoop:
 			})
 		}
 
-		// resp.Contents must have at least one element or we would have returned not found
-		listObjectsInput.Marker = resp.Contents[len(resp.Contents)-1].Key
+		listObjectsInput.ContinuationToken = resp.ContinuationToken
```

Comment:
What is `resp.ContinuationToken` set to when the initial request does not include it?

Per [AWS Docs](https://docs.aws.amazon.com/AmazonS3/latest/API/API_ListObjectsV2.html):

> ContinuationToken
If ContinuationToken was sent with the request, it is included in the response.

Reply:
same problem, I'll fix it.